format/FormatNumber.js

```diff
@@ -35,6 +35,8 @@ const UP_TICK = '▴',
  *      align vertically with negative ledgers in columns.
  * @param {boolean} [opts.withPlusSign] - true to prepend positive numbers with a '+'.
  * @param {boolean} [opts.withSignGlyph] - true to prepend an up / down arrow.
+ * @param {boolean} [opts.withCommas] - whether to include comma delimiters, defaults to true.
+ * @param {boolean} [opts.omitFourDigitComma] - true to omit delimiter if value is less than 10,000
```

Comment:
Super not in love with this name, but do like the two property approach

Reply:
I agree, and think the name is as good as it gets.  I do think that Anselm is correct in the slack conversation that it should be used only in the case of exactly 4 printed digits, that is, not if there are decimal places.